pkg/identity/webhook/webhook.go

```diff
@@ -171,11 +171,17 @@ func (h *WebhookHandler) authorizeToken(w http.ResponseWriter, r *http.Request,
 		return
 	}
 
-	allowed, reason, err := h.Authorizer.Authorize(attrs)
-	glog.V(4).Infof("<<<< authorizeToken: %v, %v, %v\n", allowed, reason, err)
-	if err != nil {
-		http.Error(w, reason, http.StatusInternalServerError)
-		return
+	var allowed authorizer.Decision
+	if h.Authorizer != nil {
+		allowed, reason, err := h.Authorizer.Authorize(attrs)
```

Comment:
you should use = instead of := here. Otherwise "allowed" value won't be seen outside of this section and always be equal 0 (i.e. DecisionDeny)
The tests fail because of this.

Reply:
Yeah, you are right, thanks!